haiku-apps/brie/brie-0.4.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="An IDE for rapid development of apps"
+DESCRIPTION="BeOS Rapid Integrated Environment (or BRIE for short) is an IDE \
+for rapid development of BeOS / OBOS / Zeta applications. All code is \
+generated in C/C++ using the BeOS API plus a few extensions."
+HOMEPAGE="https://github.com/HaikuArchives/Brie"
+COPYRIGHT="2002-2004 Sikosis"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="f3d1dadac364295d4aa84b23a41cab948b0c77be"
+SOURCE_URI="https://github.com/HaikuArchives/Brie/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="f593a2bede290ee21758985a1456e63566a83dcfbfb6b9f4319caf9e43abcb4c"
+SOURCE_FILENAME="Brie-$srcGitRev.tar.gz"
+SOURCE_DIR="Brie-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	brie = $portVersion
+	app:BRIE = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:gcc
+	cmd:make
+	"
+defineDebugInfoPackage brie \
```

Comment:
Hmm, we like to have an empty line above defineDebugInfoPackage.
Could you add it please? Thanks!